app/utils/DesResponse.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import play.api.libs.json.{JsValue, Json}
+
+
+object DesResponse {
+
+  val jsonResponse400: JsValue = Json.parse(
+    s"""
+       |{
+       | "code": "INVALID_PAYLOAD",
+       | "reason": "Submission has not passed validation. Invalid Payload."
+       |}""".stripMargin)
+
+  val jsonResponse503: JsValue = Json.parse(
+    s"""
+       |{
+       | "code": "SERVICE_UNAVAILABLE",
+       | "reason": "Dependent systems are currently not responding"
+       |}""".stripMargin)
+
+  val jsonResponse500: JsValue = Json.parse(
```

Comment:
@carrielies These error responses do not match the YAML specification. For example the structure in the YAML is
```
{
  "failures": [
    {
      "code": "INVALID_IDTYPE",
      "reason": "Submission has not passed validation. Invalid parameter idType."
    }
  ]
}
```

Reply:
These are already fixed in my latest code. Just going through the yaml now. As mentioned I was just pushing up a quick version of the stub. I'll wait until it's complete though